controllers/region.js

```diff
@@ -292,7 +292,7 @@ export const getObjRegionCids = obj => {
  * @param fields Selected region fields. Array, but in case of 'fromDb' - object
  * @param [fromDb] Select from db, not just from cache (in cahce not all fields presented)
  */
-function getObjRegionList({ obj, fields, fromDb }) {
+export async function getObjRegionList({ obj, fields, fromDb }) {
```

Comment:
ok, but this now cause a unnecessary chain of async/await in sitemap.js. May be it is actually better to split this function into two given we have two destinctive usecases and deprecate fromDb param?